### PR TITLE
Add example small molecule data set

### PR DESCRIPTION
**Pull request by benjaminhwilliams**
Includes ten image files for each of four sweeps (a subset of https://doi.org/10.5281/zenodo.51405, uncompressed) and associated processed files.
datablock.json was generated with DIALS 1.14.0-g6458063ec.
imported_experiments.json and strong.pickle were generated with DIALS 2.dev.89-gce3e9b077.

### CODE REVIEW

**Comment by Anthchirp** [comment]
do you envisage a point where having the full dataset would be useful?

File: dials_data/definitions/l_cysteine_dials_output.yml [hunk 1]
```diff
@@ -1,22 +1,68 @@
...
 license: CC-BY 4.0
 description: >
   Typical DIALS output for a small-molecule dataset.
 
-  These were derived from the DIALS small molecule tutorial L-cysteine dataset
-  (https://zenodo.org/record/51405 files l-cyst_0[1-4].tar.gz).
+  Includes ten image files for each of four sweeps (a subset of https://doi.org/10.5281/zenodo.51405,
+  l-cyst_0[1-4].tar.gz, uncompressed) and associated processed files.
 
   On a linux terminal, the relevant CBF-format image files can be obtained with
```
Comment (Anthchirp) [inline dials_data/definitions/l_cysteine_dials_output.yml]: would probably remove those three lines when images are included in the dataset.

**Comment by Anthchirp** [comment]
might make more sense to put the dataset either as a subset of the full dataset or as the full dataset into `l_cysteine` or similar, and keep the derived data separately. There may also be scope to do something more clever and add a feature to dials-data to select a subset of files, eg. ```dials_data("l_cysteine")``` for all files vs ```dials_data("l_cysteine.subset")``` for the 40 selected files, stored once on disk without duplication.

**Comment by benjaminhwilliams** [comment]
I could well imagine that having the full data set could be useful for some tests.  I now see that #12 is a plan.  I agree, let's keep the derived data in this record and have another one, pointing at https://doi.org/10.5281/zenodo.51405, pending #12.  Of course, if #12 includes an option to decompress only the desired files, the only cost is in the initial download.

I'll make the requested changes.

**Comment by benjaminhwilliams** [comment]
These changes have been superseded.  I'm re-writing the only test (in DIALS) that relied on this PR.  I think this can be safely abandoned.

**Pull request closed without merging**